src/main/sensors/boardalignment.c

```diff
@@ -64,51 +64,55 @@ static void alignBoard(int32_t *vec)
 
 void alignSensors(const int32_t *src, int32_t *dest, uint8_t rotation)
 {
-    static uint32_t swap[3];
-    memcpy(swap, src, sizeof(swap));
+    int32_t x;
```

Comment:
@ledvinap , I've got rid os `swap[]` and use a temporary - only one is required.